src/wallet/wallet_rpc_server.cpp

```diff
@@ -104,6 +104,16 @@ namespace
         entry.suggested_confirmations_threshold = std::max(entry.suggested_confirmations_threshold, (unlock_time - now + DIFFICULTY_TARGET_V2 - 1) / DIFFICULTY_TARGET_V2);
     }
   }
+  //------------------------------------------------------------------------------------------------------------------------------
+  bool is_valid_language(const std::string &language)
+  {
+    std::vector<std::string> language_list;
```

Comment:
Might make more sense for this to be an ElectrumWords method, that way you can avoid copying the language list to a temp vector.